app/src/main/java/mozilla/org/webmaker/view/WebmakerWebView.java

```diff
@@ -5,6 +5,9 @@
 import android.view.ViewGroup;
 import android.webkit.WebView;
 
+import android.webkit.JsPromptResult;
+import android.webkit.WebChromeClient;
```

Comment:
this probably requires the same extensions that mozilla.org.webmaker.client.WebClient got, come to think of it...